master/setup.py

```diff
@@ -250,7 +250,8 @@ def define_plugin_entries(groups):
         ('buildbot.secrets', [
             ('buildbot.secrets.providers.file', ['SecretInAFile']),
             ('buildbot.secrets.providers.passwordstore', ['SecretInPass']),
-            ('buildbot.secrets.providers.vault', ['HashiCorpVaultSecretProvider'])
+            ('buildbot.secrets.providers.vault', [
+                'HashiCorpVaultSecretProvider', 'HvacKvSecretProvider'])
```

Comment:
This class is in vault_hvac.py, not vault.py
```suggestion
            ('buildbot.secrets.providers.vault', [
                'HashiCorpVaultSecretProvider'],
             'buildbot.secrets.providers.vault_hvac', 
               ['HvacKvSecretProvider'])
```